modules/comment-likes.php

```diff
@@ -11,11 +11,14 @@
  * Additional Search Queries: like widget, like button, like, likes
  */
 
-Jetpack::dns_prefetch(
-	array(
-		'//widgets.wp.com',
-	)
-);
+function jetpack_dns_comment_likes_prefetch( $urls, $relation_type ) {
+	if( 'dns-prefetch' == $relation_type ) {
+		$urls[] = '//widgets.wp.com';
+	}
+	return $urls;
+}
+add_filter( 'wp_resource_hints', 'jetpack_resource_hints', 10, 2 );
```

Comment:
You will want to update the function name here and everywhere else. `jetpack_resource_hints` does not exist.

Reply:
So something along the lines of (for this example):
```suggestion
add_filter( 'wp_resource_hints', 'jetpack_dns_comment_likes_prefetch', 10, 2 );
```